templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -185,7 +205,8 @@ Resources:
 
 Outputs:
   DefaultKey:
-    Description: Default Ec2 keypair name
+    Condition: KeyProvided
+    Description: Default EC2 keypair name
```

Comment:
Keeping this in here, even though keypair is optional and cannot be checked by dependents down stream, id prefer to keep exporting it based on the `KeyProvided` conditional just in case other unknown dependents need it